lib/wargear/__init__.py

```diff
@@ -0,0 +1,120 @@
+"""
+"""
+from datetime import datetime
+import json
+import logging
+import os
+import pathlib
+from pprint import pformat
+import sqlite3
+
+from bs4 import BeautifulSoup
+from dateutil.parser import parse
+import requests
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+
+class WargearAPIClient:
+    def __init__(self, username, password):
+        self.base_url = "http://www.wargear.net"
+        self.session = requests.Session()
+        self.username = username
+        self.password = password
+
+    def authorize_session(self):
+        """Loads cookies if present on disk, otherwise authorizes by POST."""
+        cookie_jar = pathlib.Path("cookies.json")
+        if cookie_jar.is_file():
+            logger.debug("Loading cookies from disk.")
+            with cookie_jar.open("r") as file_handle:
+                cookies = json.load(file_handle)
+            self.session.cookies.update(cookies)
+            return
+
+        auth_url = self.base_url + "/player/login"
```

Comment:
Not sure if it makes a difference in Python, but for Scala/Java, string creation is slightly expensive and if it's in a method, you're recreating it each time the method is called. So I like to define strings outside of methods when possible:
```
LOGIN_URI = "/player/login"
PLAYER_INFO_URI = "/players/info"
...
```

Reply:
Good point. I also like to have any strings that might change in one place for easier updates.